cmd/manager/main.go

```diff
@@ -147,6 +148,9 @@ func main() {
 	// Add the Metrics Service
 	addMetrics(ctx, cfg)
 
+	metrics.RegisterCustomMetrics()
+	defer metrics.UnregisterCustomMetrics()
```

Comment:
@xcoulon would appreciate your input here as well

Reply:
ahem... I can't recall why I added that 😳 Maybe it avoids issues during rolling updates for metrics already registered? To be honest, I don't remember :(

Reply:
No worries, I'll do some digging around this as well

Reply:
Reading through some of the docs and issues it seems like the only real value of unregistering metrics is to reduce memory used by the registry by not tracking metrics that will no longer produce any value. I *think* an example of this is for cases where a metric is registered dynamically eg. A metric tracking some aspect of nodes where nodes can be added/removed dynamically. So I guess we can remove this line since we don't need to Unregister the metrics in this way.

Reply:
Yeah, let's remove this line :+1: